### ISSUE TITLE
Video recordings and streaming

### ISSUE THREAD

**Issue by benhylau** [dweb-camp-2019/projects#10]
I was at the Farm this past weekend and made some notes about its connectivity status and our ideas for space use:

- The current Internet access is beamed all the way from Monterey, 40 Mbps tops, but measured 5 / 12 Mbps and it broke done once at times, with dropped packets and fluctuating latencies at times, these are all non-ideal conditions for a livestream over Internet
- The local meshnet is planned with 800 Mbps bandwidth, and it looks like we can pull it off based on the high points (telephone poles and building roofs) and power access at the site
- We are thinking to use one room with a screen for lightning talks and presentations, otherwise there are no stages, but small gatherings instead (recording sessions in that room may be useful)
- The space has large areas that do not have line of sight to each other, perhaps a local livestream across the meshnet to let people know what's up on the other end may be useful (also an interesting experiment)
- Video recordings of some of these small gatherings and interviewing people may be nice for documentation purposes
- If people want to pursue ingesting the stream and publishing on a peer-to-peer protocol, perhaps the interface is for the video team to recording to an on-site RTMP / HLS server, and groups can ingest streams to then publish onto their p2p systems

I will publish more details from my site survey and kick-off meetings next week. Opening up this thread for discussion, feel free to ask me questions here or on chat!

**Comment by whanamura** [comment]
We now have Ryan Taylor who has volunteered to do Live Streaming/taping of Lightning talks.
Also Jonah Edwards of IA is working on getting 300 Mbps into the Farm using Coastal ISP and erecting  2 towers on the Farm.  So stay tuned!

**Comment by dylorb** [comment]
Excellent stuff! 

I am wondering if anyone coming to Dweb Camp has a drone and is thinking about bringing it? It would be very cool to see the site and activities from the skies.

**Comment by whanamura** [comment]
@dylorb Yes!  Mark Graham, Director of the Wayback, is bringing his drone.  If you have experience with drones, we'd love your help taping the events from above.

**Comment by dylorb** [comment]
Awesome! I'll help however I can.

**Comment by dylorb** [comment]
Hey all - 

I've been circulating this around SSB, but thought I would put it up on the github for more eyeballs. I’m an independent doc filmmaker from Canada who is coming down to Dweb Camp in July to document some of the activities. Most of my work focuses on tech, so I’m excited to come to the camp to meet people working in the decentralized web space. If you see me kicking around dweb camp with a camera, please say hello and let me know what you're up to. I'll be in the Bay Area for all of July - so if you'd like to meet up ahead of time for a coffee and a chat, I'd love to hear from you. 

Please feel free to drop me a line through SSB (I'm @dylorb), here, or at dylan.reibling at gmail.

You can check out some of my stuff on the old web at dylanreibling.com
